app/views/guitar_gallery/index.html.erb

```diff
@@ -0,0 +1,2 @@
+<h1>Guitars#index</h1>
+<p>Find me in app/views/guitars/index.html.erb</p>
```

Comment:
IMHO this file can be deleted

Reply:
It was part of the template of Maria, this file was used in Users, so I thought it was necessary 
Are you sure it is not needed? @Ron-Lavi

Reply:
Yeah it's just the legacy view, no need for that :)

Reply:
OK, I will remove it 👍🏼